src/main/webapp/script.js

```diff
@@ -81,13 +94,29 @@ async function getChart() {
 
     const chart = new google.visualization.ColumnChart(
         document.getElementById('chart-container'));
-    chart.draw(view, options);    
-
-    const averageContainer = document.getElementById('average-score-container');
-    averageContainer.innerHTML = "Average Sentiment Score: " + averageScore;
-  });
+    chart.draw(view, options); 
 }
 
+/**
+ * Create a word cloud based on the number of appearance for each word
+ * @param {Map<String:Integer>} wordFrequencyMap 
+ */
+function getWordCloudChart(wordFrequencyMap) {
+    var data = [];
+    Object.keys(wordFrequencyMap).forEach(wordKey => data.push({"x": wordKey, "value": wordFrequencyMap[wordKey]}));
+    // create a tag cloud chart
+    var chart = anychart.tagCloud(data);
+
+    // set the chart title
+    chart.title('Most Popular Comments')
+    // set array of angles, by which words will be placed
+    chart.angles([0])
```

Comment:
It is not entirely clear what the purpose of these angles are, or why it's being set to 0. Is this just part of the API you used? Maybe add more elaboration with the comments?